.github/workflows/lint.yml

```diff
@@ -79,6 +79,22 @@ jobs:
                       #       https://github.com/project-chip/connectedhomeip/issues/19169
                       #       https://github.com/project-chip/connectedhomeip/issues/22640
                       if [ "$idl_file" = './examples/all-clusters-app/all-clusters-common/all-clusters-app.matter' ]; then continue; fi
+                      if [ "$idl_file" = './examples/chef/devices/rootnode_contactsensor_lFAGG1bfRO.matter' ]; then continue; fi
```

Comment:
why? These are conformance lint tests that should generally be enabled.

Reply:
Sorry. I misunderstood this. I've rolled this back.